require-modules-from-pmr.sentinel

```diff
@@ -0,0 +1,16 @@
+import "tfconfig"
+import "strings"
+
```

Comment:
```suggestion

# NOTE: This policy hardcodes an organization name, and is only included for testing purposes.
# For real-life use, load the policy content from require-modules-from-pmr.template using
# template_file.

```

Reply:
Actually, I had not intended to include the require-modules-from-pmr.sentinel file.

Reply:
However, I might need it for the simulator

Reply:
I'm going to see if I can write out the policy from the template locally.

Reply:
Problem is that the file from template would be written to the Terraform worker container on the TFE server, not to user's laptop from which they would run `sentinel test`